tests/unit_tests/test_unit.py

```
@@ -264,6 +264,19 @@ def test_version():
     assert casanovo.__version__ is not None
 
 
+@pytest.mark.parametrize(
+    "model_file,expectation",
```

Ruff (0.15.6)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)